### PR TITLE
Added edgecmd example for every curl command in repo

### PR DESCRIPTION
**Pull request by cmorgan-osi**

### CODE REVIEW

File: content/configuration/data-selection.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ Complete the following steps to configure a Modbus TCP data selection. Use the `
     curl -d "@ConfigureDataSelection.json" -H "Content-Type: application/json" -X PUT "http://localhost:5590/api/v1/configuration/Modbus1/DataSelection"
     ```
 
+    Alternatively, use the following edgecmd command:
+
+    ```bash
+    edgecmd -cid Modbus1 set dataselection -file ./ConfigureDataSelection.json
```
Comment (aveva-jhwang) [inline content/configuration/data-selection.md:36]: This is a general comment, not specific to just this one. I'm guessing capitalization doesn't matter but can we keep the facet names consistent across our examples and documentation? I've seen "dataselection", "dataSelection", and "DataSelection" in various locations.

**Review by bballentine123 — DISMISSED** [review]
I've tested all the commands, and they all work except for the `component` instead of `components` in system-components.md

File: content/configuration/system-components.md [hunk 2]
```diff
@@ -34,6 +34,17 @@ Complete the following steps to configure system components. Use the `PUT` metho
...
 
+    Alternatively, use one of the following edgecmd commands:
+    ```bash
+    edgecmd set components -file ./ConfigureComponents.json
+    ```
+    OR
+
+    ```bash
+    # Add modbus component to existing components configuration
+    edgecmd add component -id Modbus1 -type Modbus
```
[the remark this answers is not in the document]
Comment (cmorgan-osi) [reply content/configuration/system-components.md:45]: Yes, great catch!

Comment (cmorgan-osi) [inline content/configuration/system-components.md:45]:
```suggestion
    edgecmd add components -id Modbus1 -type Modbus
```

**Pull request merged**